### ISSUE TITLE
Windows: disable QUICK_EDIT_MODE on start

### ISSUE THREAD

**Issue by ForNeVeR** [elw00d/consoleframework#66]
Currently, Windows 10 console has `ENABLE_QUICK_EDIT_MODE` flag by default which messes up any mouse interactions.

I suggest ConsoleFramework should disable this mode when starting. I've done it via this quick F# snippet:

```fsharp
let private initializeWindows() =
    let stdin = Windows.getStdInHandle()
    let consoleMode = Windows.getConsoleMode stdin
    if consoleMode &&& Windows.Constants.ENABLE_QUICK_EDIT_MODE <> 0u then
        let newMode =
            (consoleMode ||| Windows.Constants.ENABLE_EXTENDED_FLAGS) &&& ~~~Windows.Constants.ENABLE_QUICK_EDIT_MODE
        Windows.setConsoleMode stdin newMode
```

(essentially, you should just call `SetConsoleMode(stdin, (mode | ENABLE_EXTENDED_FLAGS) & ~ENABLE_QUICK_EDIT_MODE)`)

**Comment by ForNeVeR** [comment]
Looks like it was fixed by 87ae9a527fb3e1dee7d1a52338909d4429c5c5ac, right?

**Comment by igor-kostromin** [comment]
Yes, will close this, thanks !
